apps/predbat/predbat.py

```diff
@@ -910,7 +909,30 @@
         "support_charge_freeze": False,
         "support_discharge_freeze": False,
     },
+    "SE": {
+        "name": "Solar Edge",
+        "has_rest_api": False,
+        "has_mqtt_api": False,
+        "has_service_api": True,
+        "output_charge_control": "power",
+        "has_charge_enable_time": False,
+        "has_discharge_enable_time": False,
+        "has_target_soc": False,
+        "has_reserve_soc": False,
```

Comment:
Maybe a discussion to be had around allowing / encouraging this to be set to `True` when a backup interface is present?

Reply:
I've added that to the apps.yaml, this can now override the .py code